src/components/themed/WalletListFooter.tsx

```diff
@@ -6,15 +6,15 @@ import { SPECIAL_CURRENCY_INFO } from '../../constants/WalletAndCurrencyConstant
 import { useHandler } from '../../hooks/useHandler'
 import s from '../../locales/strings'
 import { useSelector } from '../../types/reactRedux'
-import { NavigationProp } from '../../types/routerTypes'
+import { NavigationBase } from '../../types/routerTypes'
 import { EdgeTokenId } from '../../types/types'
 import { WalletListModal, WalletListResult } from '../modals/WalletListModal'
 import { Airship, showError } from '../services/AirshipInstance'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 import { EdgeText } from './EdgeText'
 
 interface Props {
-  navigation: NavigationProp<'walletList'>
+  navigation: NavigationBase
```

Comment:
The more I review, the more I feel uncomfortable widening the type for these components. It's not a breaking API change, but it may be a breaking implementation change, and what's worse is that we wont see the bug until a dev mis-uses the component... what's worse is this bug may not crop up during static analysis, but instead during runtime which may even be in production. See what I mean?